shells/lib/arcs-ui/dist/install-ui-classes.js

```diff
@@ -0,0 +1,23 @@
+/**
```

Comment:
Are the files in `dist` really meant to be included in this PR? We generally shouldn't be checking in build artefacts.

Reply:
I checked in the `dist` dirs with the following reasoning:

- they are rarely changed
- the output is always exactly the same for the same input 
- trading github space for dev time is good

I realize this is not typical, and mostly I just wanted to air out this notion. I can go back to old way if that's the call.

Reply:
Counterpoints:
- The build isn't that slow, and since they're rarely changed it's an infrequent overhead.
- Adding duped files is very likely to cause confusion in reviews and when searching through code (either grep-style or in the github interface).

I would strongly prefer not to check these in.

Reply:
Ok, good enough for me. Thanks for the details.